core/templates/save-all.tid

```diff
@@ -1,6 +1,6 @@
 title: $:/core/save/all
 
 \define saveTiddlerFilter()
-[is[tiddler]] -[prefix[$:/state/popup/]] -[[$:/HistoryList]] -[[$:/boot/boot.css]] -[type[application/javascript]library[yes]] -[[$:/boot/boot.js]] -[[$:/boot/bootprefix.js]] +[sort[title]] $(publishFilter)$
+[is[tiddler]] -[prefix[$:/state/popup/]] -[prefix[$:/HistoryList]] -[[$:/boot/boot.css]] -[type[application/javascript]library[yes]] -[[$:/boot/boot.js]] -[[$:/boot/bootprefix.js]] +[sort[title]] $(publishFilter)$
```

Comment:
Can we avoid making changes to the default save filters? These are commonly customised by end users.

Reply:
Not really, if we don't want users to store huge HistoryLists ... IMO it should have been implemented with  a prefix from the beginning. ... 

TWs design should be able to handle several independent "stories". The video explains, why I think, the "tabbed-toc" ... is an independent story. ... There is huge potential in the new concept. 

Like: "In Tiddler Anchor Navigation" and "In Tiddler TOCs", which are requested quite a lot. ...

The new "focused-tiddler" mechanism will enable all of those possibilities. ... Which the video shows.